### PR TITLE
migrate downsampling apis to new template

### PR DESCRIPTION
**Pull request by charislam**
# Description

Migrate downsampling API references to new format

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/_hyperfunctions/asap_smooth.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  version:
+    experimental: 0.2.0
+    stable: 1.11.0
+hyperfunction:
+  family: downsampling
+  type: function
+api_details:
+  summary: >
+    Downsample your data with the [ASAP smoothing algorithm](https://arxiv.org/pdf/1703.00983.pdf).
+    This algorithm preserves the rough shape and larger trends of the input data, while minimizing the local variance between points.
```
Comment (Loquacity) [inline api/_hyperfunctions/asap_smooth.md:18]: I wonder if "rough shape" translates poorly?

File: api/_hyperfunctions/asap_smooth.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    returns:
+      - column: asap_smooth
+        type: Timevector
+        description: >
+          An object representing a series of values occurring at set intervals from a starting time.
+          It can be unpacked with `unnest`.
+          For more information, see the documentation on [timevectors](/timescaledb/:currentVersion:/how-to-guides/hyperfunctions/function-pipelines/#timevectors).
+  examples:
+    - description: >
+        This example uses a table called metrics, with columns for date and reading that contain measurements that have been accumulated over a large interval of time.
```
Comment (Loquacity) [inline api/_hyperfunctions/asap_smooth.md:47]:
```suggestion
        This example uses a table called `metrics`, with columns for `date` and `reading`. The columns contain measurements that have been accumulated over a large interval of time.
```

**Review by Loquacity — APPROVED** [review]
Just a couple of non-blocking things.

File: api/_hyperfunctions/asap_smooth.md (hunk 1 above)
Comment (charislam) [reply api/_hyperfunctions/asap_smooth.md:18]:
```suggestion
    This algorithm preserves the approximate shape and larger trends of the input data, while minimizing the local variance between points.
```

**Pull request merged**